cmd/clusterctl/client/cluster/cert_manager.go

```diff
@@ -347,13 +341,9 @@ func (cm *certManagerClient) shouldUpgrade(objs []unstructured.Unstructured) (st
 		// if there is no version annotation, this means the obj is cert-manager v0.11.0 (installed with older version of clusterctl)
 		objVersion, ok := obj.GetAnnotations()[clusterctlv1.CertManagerVersionAnnotation]
 		if !ok {
-			// try the old annotation name
-			objVersion, ok = obj.GetAnnotations()[certManagerVersionAnnotation]
-			if !ok {
-				currentVersion = "v0.11.0"
-				needUpgrade = true
-				break
-			}
```

Comment:
The logic after this commit has changed though; now we're always defaulting to `v0.11.0` if the annotation isn't there; which might be ok. @fabriziopandini is this what we want to do?